CHANGELOG/CHANGELOG-3.5.md

```diff
@@ -7,30 +7,30 @@ Previous change logs can be found at [CHANGELOG-3.4](https://github.com/etcd-io/
 ## v3.5.8 (TBD)
 
 ### etcd server
-- Fix [server/embed: fix data race when starting both secure & insecure gRPC servers on the same address](https://github.com/etcd-io/etcd/pull/15517)
+- Fix [Watch response traveling back in time when reconnecting member downloads snapshot from the leader](https://github.com/etcd-io/etcd/pull/15515)
+- Change [http2 frame scheduler to random algorithm](https://github.com/etcd-io/etcd/pull/15452)
+- Fix [race when starting both secure & insecure gRPC servers on the same address](https://github.com/etcd-io/etcd/pull/15517)
 - Add [`etcd --tls-min-version --tls-max-version`](https://github.com/etcd-io/etcd/pull/15483) to enable support for TLS 1.3.
 - Fix [server/auth: disallow creating empty permission ranges](https://github.com/etcd-io/etcd/pull/15619)
-
-### Package `client/pkg/v3`
+- Upgrade [golang.org/x/net to v0.7.0](https://github.com/etcd-io/etcd/pull/15337)
+- Bump bbolt to [v1.3.7](https://github.com/etcd-io/etcd/pull/15222).
 - Fix [aligning zap log timestamp resolution to microseconds](https://github.com/etcd-io/etcd/pull/15240). Etcd now uses zap timestamp format: `2006-01-02T15:04:05.999999Z0700` (microsecond instead of milliseconds precision).
+- Add [`etcd --listen-client-http-urls`](https://github.com/etcd-io/etcd/pull/15589) flag to support separating http server from grpc one, thus giving full immunity to [watch stream starvation under high read load](https://github.com/etcd-io/etcd/issues/15402).
 
 ### Package `netutil`
 - Fix [consistently format IPv6 addresses for comparison](https://github.com/etcd-io/etcd/pull/15187)
 
 ### Package `clientv3`
 - Fix [etcd might send duplicated events to watch clients](https://github.com/etcd-io/etcd/pull/15274).
 
-### Dependency
-- Bump bbolt to [v1.3.7](https://github.com/etcd-io/etcd/pull/15222).
+### Go
+- Require [Go 1.19+](https://github.com/etcd-io/etcd/pull/15337).
+- Upgrade [go to 1.19.7](https://github.com/etcd-io/etcd/pull/15429)
```

Comment:
```suggestion
- Upgrade [go to 1.19.7](https://github.com/etcd-io/etcd/pull/15428)
```

Or change it to 1.19.8 as I bumped to 1.19.8 in https://github.com/etcd-io/etcd/pull/15651.

I can update it in a separate PR if you want to keep it as it's for now.

Reply:
done